R/utils.R

```diff
@@ -153,3 +153,91 @@ near <- function(x, y) {
   tol <- .Machine$double.eps^0.5
   abs(x - y) < tol
 }
+
+
+# creates a wrapper function around primitive functions
+wrap_primitive <- function(fun) {
```

Comment:
This seems to be unused?
It can stay, but I was wondering what you were thinking of using it for.

Reply:
i used it in a previous version, now no longer necessary